ios/FluentUI/Command Bar/CommandBarButtonGroupView.swift

```diff
@@ -20,13 +20,23 @@ class CommandBarButtonGroupView: UIView {
 
         configureHierarchy()
         applyInsets()
+        hideGroupIfNeeded()
     }
 
     @available(*, unavailable)
     required init?(coder: NSCoder) {
         preconditionFailure("init(coder:) has not been implemented")
     }
 
+    /// Update the passed button and hide the group view if requested
+    func update(_ button: CommandBarButton, updateGroupView: Bool) {
```

Comment:
something about this make me feel messy. this is meant to be each button need to update. not all property update tirggers group stackview to be hidden. rather than pipling all of propertyChangedUpdateBlock to trigger this, can we just call the hideGroupIfneeded() when the hidden property changes?